qiita_pet/handlers/base_handlers.py

```diff
@@ -4,38 +4,34 @@
 
 
 class BaseHandler(RequestHandler):
-    def user(self):
-        if self.current_user:
-            return User(self.current_user)
-        return None
-
     def get_current_user(self):
         '''Overrides default method of returning user curently connected'''
-        user = self.get_secure_cookie("user")
-        if user is None:
+        username = self.get_secure_cookie("user")
+        if username is not None:
+            username = username.strip("\"' ")
```

Comment:
It returns the username with quotes around it when you use the cookie function, so I strip them off here.